WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -35,8 +36,9 @@ struct ShippingLabelPackageList: View {
                             SelectableItemRow(title: package.title,
                                               subtitle: package.dimensions + " \(viewModel.dimensionUnit)",
                                               selected: selected).onTapGesture {
-                                viewModel.didSelectPackage(package.id)
-                            }
+                                                viewModel.didSelectPackage(package.id)
+                                                ServiceLocator.analytics.track(.shippingLabelPurchaseFlow, withProperties: ["state": "packages_selected"])
```

Comment:
Just to confirm, do we want to fire this event every time a package is selected in the list on the Package Selected screen (current behavior)?

Or do we only want to fire it once, when the selection is confirmed (i.e. when the "Done" button is tapped)?

Reply:
Yes, because we select the package when the user taps the package, and they can dismiss the view without pressing done. So, `Done` on this screen works like a back button (since it's a modal view).